src/Sulu/Bundle/ActivityBundle/SuluActivityBundle.php

```diff
@@ -18,10 +18,13 @@
 use Symfony\Component\DependencyInjection\Extension\ExtensionInterface;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
-class SuluActivityBundle extends Bundle
+final class SuluActivityBundle extends Bundle
 {
     use PersistenceBundleTrait;
 
+    /**
+     * @internal
```

Comment:
we should add the same comment as here to be more transparent: https://github.com/sulu/sulu/blob/c785900f346166d01304fbd6023ee4a106128550/packages/article/src/Infrastructure/Symfony/HttpKernel/SuluArticleBundle.php#L91